charts/ocis/values.yaml

```
@@ -86,25 +86,26 @@ insecure:
 
 cache:
   # -- Type of the cache to use.
-  # To disable the cache, set to "noop".
-  # Can be set to "redis-sentinel", "noop", "nats-js-kv" or "memory" (only for testing) then the address of Redis (Sentinel) / etcd / nats-js node(s) needs to be set to `cache.nodes`.
+  # Can be set to "nats-js-kv" or "redis-sentinel". There are also the not recommended options "memory" and "noop".
```

I would write it this way. Not sure which is more correct.

```suggestion
  # Can be set to "nats-js-kv" or "redis-sentinel". There are also the non-recommended options "memory" and "noop".
```